test/extended/builds/pipeline.go

```diff
@@ -159,24 +169,139 @@ var _ = g.Describe("[Feature:Builds][Slow] openshift pipeline build", func() {
 			dcLogFollow, dcLogStdOut, dcLogStdErr, err = oc.Run("logs").Args("-f", "dc/jenkins").Background()
 			o.Expect(err).NotTo(o.HaveOccurred())
 
-		}
-	)
-
-	g.Context("Pipelines with maven slave", func() {
-		g.BeforeEach(func() {
-			setupJenkins(jenkinsEphemeralTemplatePath)
-
 			if os.Getenv(jenkins.EnableJenkinsMemoryStats) != "" {
 				ticker = jenkins.StartJenkinsMemoryTracking(oc, oc.Namespace())
 			}
 
 			g.By("waiting for builder service account")
-			err := exutil.WaitForBuilderAccount(oc.KubeClient().Core().ServiceAccounts(oc.Namespace()))
+			err = exutil.WaitForBuilderAccount(oc.KubeClient().Core().ServiceAccounts(oc.Namespace()))
 			o.Expect(err).NotTo(o.HaveOccurred())
+		}
+	)
+
+	g.Context("jenkins-client-plugin tests", func() {
```

Comment:
please add a comment explaining why these are their own bucket, lest someone like me get the bright idea to fold them into the other emphemeral-template based tests, like i almost did while reviewing this before remembering the comment you made in the PR about needing them separate :)